Doc/library/os.rst

```diff
@@ -1328,9 +1350,10 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
 
    Write the contents of *buffers* to file descriptor *fd*. *buffers* must be a
    sequence of :term:`bytes-like objects <bytes-like object>`. Buffers are
-   processed in array order. Entire contents of first buffer is written before
-   proceeding to second, and so on. The operating system may set a limit
-   (sysconf() value SC_IOV_MAX) on the number of buffers that can be used.
+   processed in array order. Entire contents of the first buffer is written
+   before proceeding to the second, and so on. The operating system may set a
+   limit (sysconf() value SC_IOV_MAX) on the number of buffers that can be
```

Comment:
It will make sense to change 

 sysconf() -> :func:`sysconf` 

and

SC_IOV_MAX ->  ``'SC_IOV_MAX'``

?

Reply:
Oh wait, I didn't want to modify this documentation... But then I read the doc, and oh, all these documentations can be enhanced...